### PR TITLE
Provide an example of FreeIPA configuration

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

**Comment by domcleal** [comment]
@eLobato could you confirm this please?

File: _includes/manuals/1.5/4.1.1_authentication.md [hunk 1]
```diff
@@ -51,6 +49,23 @@ attr lastname     = sN
...
 
+##### FreeIPA
+
+<pre>
+Name              = My Directory
+Host              = host.domain.org
+Port              = 636
+TLS               = yes
+Onthefly register = yes
+Account           = uid=$login,cn=users,cn=accounts,dc=example,dc=org
```
Comment (dLobatog) [inline _includes/manuals/1.5/4.1.1_authentication.md:60]: Just $login should be enough

**Comment by dLobatog** [comment]
OK except for the whole user DN which shouldn't be necessary. Can you update the configuration on top of the file too (after 'Click on New Authentication Source and enter the following')? It has changed a little now.

**Comment by stbenjam** [comment]
@eLobato Shouldn't be needed, but couldn't get it to work without it on FreeIPA 3 or 4 for some reason.

**Comment by domcleal** [comment]
I think it'll depend on whether you're using 1.5 or 1.6.  1.6 with ldap_fluff has an optional DN, but I don't think 1.5 would have.  I'll merge on that basis and will copy + adjust for 1.6 soon, once I've fixed the remaining bugs...

**Comment by domcleal** [comment]
Thanks @stbenjam

**Pull request merged**